### ISSUE TITLE
How to only allow single file upload?

### ISSUE THREAD

**Issue by imkven** [balderdashy/skipper#97]
I want to limit file upload, but I can't find any options for it. I have try below code:

```javascript
req.files // Always empty
```
to detect how many files upload.

**Comment by metzuda** [comment]
Single file : `<input type="file" name="img">`
Multiple files : `<input type="file" name="img" multiple>`

**Comment by imkven** [comment]
Thanks, @metzuda 

But it not working too. User still can POST multiple files to server. I guess the only way to fix this is checking how many files are passing to server.

**Comment by sailsbot** [comment]
Thanks for posting, @imkven.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
